src/logic/schedule-logic/children-info.logic.ts

```diff
@@ -3,16 +3,12 @@ import { BaseSectionLogic } from "./base-section-logic.model";
 import { ChildrenSectionKey } from "../section.model";
 import { ChildrenInfoProvider } from "../providers/children-info-provider.model";
 import { Sections } from "../providers/schedule-provider.model";
+import { Omit } from "@material-ui/core";
 
-export class ChildrenInfoLogic extends BaseSectionLogic implements ChildrenInfoProvider {
```

Comment:
@pectom 
Usunęliśmy dziedziczenie po `BaseSectionLogic`, ponieważ `BaseSectionLogic` wymaga `sectionKey`, który musi należeć do interfejsu `Sections`. A do tego interfejsu z zalożenia należą tylko te sekcje, co są widoczne.

 Nie mniej jednak, myśle, że potrzeba będzie w jakiś sposób to przekminić, ponieważ `BaseSectionLogic` ma metodę `updateSection`, a my czasami chcemy zmieniać liczbę dzieci i było by fajnie użyć metody która jest zdefiniowana na `BaseSectionLogic` zamiast duplikować kod.